IMU: 
  VelocityRW: 0.008
  AccBiasRW: 0.0001
  AccBiasInitStd: 0.02
  AttitudeRW: 0.003
  GyrBiasRW: 0.000618
  GyrBiasInitStd: 0.0175
  BrBMx: 0.0
  BrBMy: 0.0 
  BrBMz: 0.0 
  qMBw: 1.0
  qMBx: 0.0
  qMBy: 0.0
  qMBz: 0.0
Tag: 
  PixelStd: 2.0
  BrBVx: -0.0116603
  BrBVy: -0.0570516
  BrBVz: -0.00124568
  qVBw: 0.70136
  qVBx: 0.0083632
  qVBy: -0.00390811
  qVBz: 0.712747
  OutlierThreshold: 15.51
  tagSize: 0.16
States: 
  pos: 
    initx: 0.0
    inity: 0.0
    initz: 0.0
    initStdx: 0.1
    initStdy: 0.1
    initStdz: 0.1
    preNoi: 0.0001
  vel: 
    initx: 0.0 
    inity: 0.0 
    initz: 0.0 
    initStdx: 0.1
    initStdy: 0.1
    initStdz: 0.1
  att: 
    initw: 1.0 
    initx: 0.0 
    inity: 0.0 
    initz: 0.0 
    initStdx: 0.1
    initStdy: 0.1
    initStdz: 0.1
  acb: 
    initx: 0.0 
    inity: 0.0 
    initz: 0.0
  gyb: 
    initx: 0.0 
    inity: 0.0 
    initz: 0.0
  IrIT: 
    initx: 0.0 
    inity: 0.0 
    initz: 0.0 
    initStdx: 10.0 
    initStdy: 10.0 
    initStdz: 10.0 
    preNoi: 0.0001
  qTI: 
    initw: 1.0 
    initx: 0.0 
    inity: 0.0 
    initz: 0.0 
    initStdx: 1.0 
    initStdy: 1.0 
    initStdz: 1.0 
    preNoi: 0.0001
    